### DISCUSSION TITLE
Having Overflow and visibility issue on MDN Playground

### DISCUSSION THREAD

**Discussion by afajansari** [mdn/mdn-community#749] [Platform] [answered]
![image](https://github.com/user-attachments/assets/353741f0-67e7-4180-9e12-31be1b6b21ff)
![image](https://github.com/user-attachments/assets/a5fcf9fc-cdf3-4fb7-92a6-69bd22275ef4)

Hi Community Members,

I started using MDN Playground since start of this year. Most of the time for learning new things on the web technologies .
From some day earlier I suddenly facing issue to MDN Playground overflowing editor box and editor content visibilty.

Can anyone help me on this. is someone facing same. Or is it due to some extensions in chrome?

Regards,
Mohammad Afaj Ansari

**Comment by caugner** [comment, marked as answer]
Hi,

Thanks for reporting, I have [created an issue](https://github.com/mdn/yari/issues/11990) for it. We'll follow up there.

Thanks!
